src/zspec/zcl/definition/cluster.ts

```diff
@@ -3447,22 +3447,58 @@ export const Clusters: Readonly<Record<ClusterName, Readonly<ClusterDefinition>>
         ID: 0x0704,
         attributes: {},
         commands: {
+            requestTunnel: {
+                ID: 0,
+                response: 0,
+                parameters: [
+                    {name: 'protocolID', type: DataType.ENUM8},
```

Comment:
```suggestion
                    {name: 'protocolId', type: DataType.ENUM8},
```

To keep it consistent with other `Id`s in this file (also change other occurrences below)

Reply:
There was `tunnelID` in description of `transferData` command already, it was added before my PR.
Should I change only `protocolID` to `protocolId`, because `tunnelID` key is used by easyiot.ts converter - and I was following this notation as an example.

Reply:
Let's also update `easyiot.ts` for consistency purposes.